shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Shelley.hs

```diff
@@ -73,6 +73,8 @@ instance CryptoClass.Crypto c => UsesTxOut (ShelleyEra c) where
 instance CryptoClass.Crypto c => UsesPParams (ShelleyEra c) where
   mergePPUpdates _ = updatePParams
 
+type instance E.TranslationContext (ShelleyEra c) = ()
```

Comment:
This is somewhat questionable, since we never actually use `TranslationContext` for `ShelleyEra`. But that's also the reason that it's harmless. And it currently simplifies Consensus. Also, it's somewhat consistent: every era with the Shelley mega-era now has a `TranslationContext` instance. (Which is trivial prior to Alonzo.)